Makefile

```diff
@@ -29,6 +29,7 @@ build-latest:
 
 build-testing:
 	$(MAKE) build TAG=testing \
+		XDR_REF=v20.0.1 \
```

Comment:
as part of reconciliate, can the core ref on soroban-dev for ci build bump also:
https://github.com/stellar/quickstart/blob/bemused-inauguratory/.github/workflows/build-soroban-dev.yml#L38
seems like that should be 20.0.1 also correct?

Reply:
I didn't change it because the soroban-dev CI yml is set to v20.0.0.

Reply:
yes, but wasn't sure why, seems like soroban-dev should be on track with 20.0.1 correct, that's what is on futurenet?

Reply:
Futurenet, according to https://horizon-futurenet.stellar.org, has v20.0.0.

Reply:
ok, I guess the 20.0.1 was initially material to testnet, and it hasn't be updated there yet, thanks.